### PR TITLE
Make FetcherReachedHeadEvent an Event

### PR DESCRIPTION
**Pull request by schmir-at-bb**
Also add some more fields, like the block time and if it originated
from the home or the foreign chain. Add an enum ChainRole for that.

### CODE REVIEW

File: tools/bridge/bridge/events.py [hunk 1]
```diff
@@ -13,6 +14,18 @@ class Event(ABC):
 Event.register(AttributeDict)
 
 
+class ChainRole(Enum):
+    home = "home"
```
Comment (jannikluhn) [inline tools/bridge/bridge/events.py]: Usually enum entries are `UPPER_CAPS` in Python, no?

File: tools/bridge/bridge/events.py [hunk 2]
```diff
@@ -13,6 +14,18 @@ class Event(ABC):
...
+    foreign = "foreign"
+
+
+@attr.s(auto_attribs=True)
+class FetcherReachedHeadEvent:
```
Comment (jannikluhn) [inline tools/bridge/bridge/events.py]:
```suggestion
class FetcherReachedHeadEvent(Event):
```

Comment (schmir-at-bb) [reply tools/bridge/bridge/events.py]: right. I missed that.

File: tools/bridge/bridge/events.py (hunk 1 above)
Comment (schmir-at-bb) [reply tools/bridge/bridge/events.py]: I don't know. I think it's ugly, but I don't dare to look it up.

**Pull request merged**